examples/square/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "square"
```

Comment:
let's prefix this so it is a bit more globally unique.
like think of some prefix for all the wasi components for layer and use that.
(Most packages are layer-*, the avs stuff for the chain is lavs-*, you can also use lavs-* or something else if you prefer... lwasi-*???)

layer-wasi-square